### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by joebayles**
Added Projects using this Repository, per https://github.com/Esri/joint-military-symbology-xml/issues/206.

### CODE REVIEW

**Comment by csmoore** [comment]
Looks like this never got assigned so assigning to @abouffard

**Comment by csmoore** [comment]
& assigning to me to verify now..

File: README.md [hunk 1]
```diff
@@ -1,7 +1,13 @@
 # joint-military-symbology-xml
 Joint Military Symbology XML (JointMilSyML or JMSML) is an XML schema, and associated instance data, designed to definitively document the contents of MIL-STD 2525D and NATO STANAG APP-6(C).
 
-___Check out this [Symbology Explorer](http://kjellmf.github.io/military-symbology-explorer/#/home) written by @kjellmf . It's a perfect example of how open source development of the JMSML benefits everyone!___
+## Projects using this Repo
+
+|Project|Repository|Website|
```
Comment (csmoore) [inline README.md]: Can you move this under the resources section & add the 2 other project names/cells "milsymbol" & "joint military symbology explorer"

& it should be good to merge.

**Comment by csmoore** [comment]
One other minor comment - this was created as a branch in this/Esri repo versus a branch in your fork so I just wanted to make sure that was intentional & how you all are handling PRs

**Comment by joebayles** [comment]
https://github.com/Esri/joint-military-symbology-xml/commit/55ddf44c512759327e7a992102b9bf4c9ae30963 based on comments from @csmoore.

**Pull request merged**